test-cases/tests/upgrade/n02-upgrade-rhmi.md

```diff
@@ -52,21 +40,28 @@ Note: This test includes all steps to prepare the cluster before the upgrade, tr
    make local/deploy
    ```
 
-   See step 10 and 11, you might want to do these pre-upgrade as well.
+   There should be no errors in the command output and product (AMQ, 3scale, SSO) URLS should not be blank. Alternatively, you can check the `Environment` tab in workload-webapp namespace in OpenShift console. See step 8 and 9, you might want to do these pre-upgrade as well.
 
-5. Edit RHMIConfig in the `redhat-rhmi-operator` config again to start the upgrade
+3. Edit RHMIConfig in the `redhat-rhmi-operator` config to start the upgrade
 
    ```
    oc edit RHMIConfig rhmi-config -n redhat-rhmi-operator
    ```
 
-6. Edit following fields in the **rhmi-config** and save:
+4. Edit following fields in the **rhmi-config** and save:
+
+   - spec.upgrade.notBeforeDays: 0
+   - spec.upgrade.waitForMaintenance: `false`
 
-   - spec.upgrade.alwaysImmediately: `true`
+   > The upgrade should start shortly. Have a look at `status.upgrade.scheduled.for`. In rare situations it might get scheduled more that 6 hours in past, in that case upgrade won't be triggered. Play with the `spec.maintenance.*` and `spec.upgrade.*` values to get it scheduled some other time.
 
-   > The upgrade should start automatically
+   Use the command below to check whether the installPlan exists and is approved. The operator should approve the installPlan based on **rhmi-config**. If the installPlan is not approved shortly, restart the rhmi-operator (delete the pod or scale down to 0 and then scale back up to 1).
```

Comment:
I understand approving the installPlan manually can cause issues, so would just be a bit more explicit;

```suggestion
   Use the command below to check whether the installPlan exists and is approved. The operator should approve the installPlan based on **rhmi-config**. The installPlan should not be approved manually - if the installPlan is not approved shortly, restart the rhmi-operator (delete the pod or scale down to 0 and then scale back up to 1).
```